src/uu/touch/src/touch.rs

```diff
@@ -41,27 +41,31 @@ pub mod options {
 
 static ARG_FILES: &str = "files";
 
-// Convert a date/time to a date with a TZ offset
-fn to_local(tm: time::PrimitiveDateTime) -> time::OffsetDateTime {
-    let offset = match time::OffsetDateTime::now_local() {
-        Ok(lo) => lo.offset(),
-        Err(e) => {
-            panic!("error: {e}");
-        }
-    };
-    tm.assume_offset(offset)
+mod format {
+    pub(crate) const POSIX_LOCALE: &str = "%a %b %e %H:%M:%S %Y";
+    pub(crate) const ISO_8601: &str = "%Y-%m-%d";
+    // "%Y%m%d%H%M.%S" 15 chars
```

Comment:
Somehow I don't see what the purpose is of this and the other similar comments.

Reply:
It's used in this function:

https://github.com/uutils/coreutils/blob/1037304cb7c0093c777928cd6242d1ad9826f694/src/uu/touch/src/touch.rs#L401-L415

It's mostly just a leftover from before, when it was harder to determine the length  of the format due to the format strings from the `time` crate. I can remove it if you want.